core/domain/subtopic_page_services.py

```diff
@@ -90,8 +90,9 @@ def get_subtopic_pages_with_ids(topic_id, subtopic_ids):
         subtopic_page_ids)
     subtopic_pages = []
     for subtopic_page_model in subtopic_page_models:
-        subtopic_pages.append(
-            get_subtopic_page_from_model(subtopic_page_model))
+        if subtopic_page_model is not None:
```

Comment:
Hi @aks681, can you review this change? (As it was causing attribute error & now if subtopic_page_model is None it will not go for finding its subtopic_page)

Reply:
Is the return models (`subtopic_pages`) supposed to match in length with input `subtopic_ids` list to function? In which case, it might be better to append `None` to `subtopic_pages` here instead of ignoring it entirely.

Reply:
Okay, will change it. Thanks!

Reply:
Changes done